src/Tasks/Microsoft.NET.Build.Tasks/targets/Microsoft.NET.PreserveCompilationContext.targets

```diff
@@ -6,15 +6,24 @@ WARNING:  DO NOT MODIFY this file unless you are knowledgeable about MSBuild and
           created a backup copy.  Incorrect changes to this file will make it
           impossible to load or build your projects from the command-line or the IDE.
 
-Copyright (c) .NET Foundation. All rights reserved. 
+Copyright (c) .NET Foundation. All rights reserved.
 ***********************************************************************************************
 -->
 <Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
 
+  <!--
+    Optional targets used to initialize PreserveCompilationContext.
+    This property is configured by the RazorSDK if it is referenced by the app.
+  -->
+  <Import Project="$(_NETSdkInitializePreserveCompilationTargets)" Condition="'$(_NETSdkInitializePreserveCompilationTargets)' != ''" />
```

Comment:
I could make this a proper `ImportBeforeMicrosoftNetSdkPreserveCompilationContext` property if we that's more appropriate. I used a "private" property since I don't really expect user projects to ever want to configure this